docs/administering/OFFBOARDING_A_FUND.md

```diff
@@ -0,0 +1,170 @@
+end
+starting_counts['fund_agnostic'] = fund_counts
+
+#save starting counts to file
+$stdout = File.new('initail_state.out', 'w')
+$stdout.sync = true
+starting_counts
+$stdout = STDOUT
+```
+
+Remove tenant-specific rows.
+```ruby
+#Lock our tenant!
+Fund.all.pluck(:name,:id)
+defunct_fund_id = #the fund you are removing
+ActsAsTenant.current_tenant = Fund.find(defunct_fund_id)
+
+Patient.destroy_all
+ArchivedPatient.destroy_all
+[Note, Fulfillment, ExternalPledge, PracticalSupport, Call].each { |model| model.destroy_all }
+[Clinic, Line, Config].each { |model| model.destroy_all }
+User.destroy_all
+
+# exit and restart to console to drop the ActsAsTenant.current_tenant
+exit
+```
+
+
+Check everything looks right before destroying the PaperTrail versions
+
+Run `rails c`
+```ruby
+# Check the new overall state
+models_fund_dependent = [ PledgeConfig, PracticalSupport, Patient, Note, Line, Fulfillment, ExternalPledge, Event, Config, Clinic, CallListEntry, Call, ArchivedPatient, User]
+models_fund_agnostic = [ Fund, ActiveRecord::SessionStore::Session, PaperTrail::Version, PaperTrailVersion] 
+
+post_deletion_counts = {}             
+Fund.all.sort.each do |fund|          
+  fund_counts = {}                    
+  models_fund_dependent.each do |model|
+    fund_counts[model.to_s.to_sym] = model.where(fund_id: fund.id).size
+  end                                                                  
+  post_deletion_counts[fund.name] = fund_counts                        
+end                                                                    
+fund_counts = {}                                                       
+models_fund_agnostic.each do |model|                                   
+  fund_counts[model.to_s.to_sym] = model.all.size                      
+end                                                                    
+post_deletion_counts['fund_agnostic'] = fund_counts                    
+                                                                       
+#save starting counts to file                                          
+$stdout = File.new('post_deletion.out', 'w')                           
+$stdout.sync = true                                                    
+post_deletion_counts                                                   
+$stdout = STDOUT
+
+exit
+```
+
+diff those files!
+
+  - The other fund objects should not have changed significantly
+  - There should be a jump in PaperTrails corresponding to the total object count we've deleted
+
+`diff -y initial_count.out post_deletion.out`
+
+IF AND ONLY IF everything looks good, let's continue.
+
+Remove the PaperTrails associated with the fund, and finally the Fund row itself.
+
+run `rails c` again
+
+```ruby
+# delete the PaperTrails
+#  🚨 ⚠️ 🚨 ⚠️ 🚨 This is the point of no return.
+# Once these are deleted, we cannot restore the funds information from this database
+defunct_fund_versions = PaperTrail::Version.where_object(fund_id: id)
```

Comment:
need to redeclare fund_id here if we're going to pop in and out of the console

Reply:
```suggestion
defunct_fund_versions = PaperTrail::Version.where_object(fund_id: defunct_fund_id)
```